scripts/monitoring/cron-send-user-running-pod-count.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+
+COUNT=$(oadm manage-node --list-pods --selector=type=compute | grep Running | wc -l)
+echo
+echo "Number of User Running pod account : $COUNT"
```

Comment:
Can we get output from this script only on failure or -v/--verbose is passed?

That way we do not get spammed from cron.